server/models/organization.js

```diff
@@ -16,10 +16,25 @@ const organizationSchema = new mongoose.Schema({
         type: String,
         required: true
     },
+    phonenumber: {
+        type: String
+    },
     image: {
         type: String,
         default: 'http://www.placekitten.com/200/200'
-    }
+    },
+    user: [
+        {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'User'
+        }
+    ],
+    event: [
+        {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'Event'
+        }
+    ]
```

Comment:
If we switch to 1:M relationship thing, then we won't need an event array here, it'll be like the Museum and it's Pieces in the codealong.